### ISSUE TITLE
Installing Avideo Encoder on Ubuntu

### ISSUE THREAD

**Issue by bee-media** [WWBN/AVideo-Encoder#514]
Help understand.
I do everything according to the instructions, on Ubuntu 20.04
When installing, the inscription PROCESS hangs and that's it. There are no errors in the log file.

**Comment by DanielnetoDotCom** [comment]
Must have something in the logs

check the apache log, or check the JS console for errors

**Comment by bee-media** [comment]
Understood this.
But when considering YPTSocket (YPTSocket), the site crashes and on the main page says Please configure your webSiteRootPath
I can't understand why.

**Comment by DanielnetoDotCom** [comment]
In your videos/configuration.php file change it

```php
$global['webSiteRootPath'] = '';
if(empty($global['webSiteRootPath'])){
    preg_match('/https?:\/\/[^\/]+(.*)/i', $global['webSiteRootURL'], $matches);
    if(!empty($matches[1])){
        $global['webSiteRootPath'] = $matches[1];
    }
}
```

to this
```php
$global['webSiteRootPath'] = '/';
```

**Comment by bee-media** [comment]
![Screenshot_1](https://github.com/WWBN/AVideo-Encoder/assets/25938240/f712bdc0-4860-4580-ae09-24f7b6c7e885)
![Screenshot_2](https://github.com/WWBN/AVideo-Encoder/assets/25938240/1222f3f7-1c69-4cc3-ae07-a2b15673be6e)
Did not help.
And this happens only after turning it on YPTSocket (YPTSocket)

**Comment by bee-media** [comment]
![Screenshot_3](https://github.com/WWBN/AVideo-Encoder/assets/25938240/11bb87b1-8284-4e52-b18b-3bdc9c49a232)

**Comment by DanielnetoDotCom** [comment]
please read carefully

https://github.com/WWBN/AVideo-Encoder/issues/514#issuecomment-1646659036

**Comment by bee-media** [comment]
Sorry, not there has changed.
But it still didn't help.
![Screenshot_4](https://github.com/WWBN/AVideo-Encoder/assets/25938240/b526262f-23d1-4524-a4ac-764c6da990f6)

**Comment by DanielnetoDotCom** [comment]
remove the lines below

if(empty($global['webSiteRootPath'])){
    preg_match('/https?:\/\/[^\/]+(.*)/i', $global['webSiteRootURL'], $matches);
    if(!empty($matches[1])){
        $global['webSiteRootPath'] = $matches[1];
    }
}

**Comment by bee-media** [comment]
It worked, thanks.
But now I don't understand why it's not streaming.
![11](https://github.com/WWBN/AVideo-Encoder/assets/25938240/fa31904b-466d-4025-8f52-95bd742df725)
![22](https://github.com/WWBN/AVideo-Encoder/assets/25938240/f62f5fc9-6684-4faa-b029-25d7789a2270)

**Comment by DanielnetoDotCom** [comment]
Open a new issue and provide more details, avideo.log and js log

**Comment by bee-media** [comment]
ok, this topic can be closed.
